tests/queries/0_stateless/03409_coalescing_replicated_merge_tree.sql

```diff
@@ -0,0 +1,53 @@
+-- Tags: replica
```

Comment:
```suggestion
-- Tags: replica, no-shared-merge-tree, no-distributed-cache
-- Tag no-shared-merge-tree: Requires update in private (@jkartseva)
-- Tag no-distributed-cache: Requires update in private (@jkartseva)
```

Reply:
The private branch still blocks it. Let's add some tags to prevent these tests from running, I'll update the private branch after this PR is merged.